karma.conf.js

```diff
@@ -1,8 +1,9 @@
 module.exports = function (config) {
   config.set({
     browsers: ['ChromeHeadless'],
-    basePath: 'target/shadow',
-    files: ['cljs-test.js'],
+    files: [
+      'node_modules/jquery/dist/jquery.min.js', // XXX: workaround for unit tests expecting js/$
```

Comment:
A CLJS unit test probably shouldn't be using jquery for anything? :thinking:

Reply:
i think this was a case of `rems.administration.test-create-form` doing side effects in re-frame events. those events probably should not do that (instead use co-effects as re-frame suggests).

nevertheless, rolled back this change since the improved async focus was not needed anymore. current implementation does not take into account elements that are in DOM but not rendered. maybe this will come back another time